src/util/nice-time.test.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-env jest */
+
+import niceTime from 'src/util/nice-time'
+
+describe('niceTime tests', () => {
+    test('should return now for timeperiod of less than 90 seconds', () => {
+        const date = new Date()
+        expect(niceTime(date)).toBe('now')
+    })
+
+    test('should return minutes for timeperiod of less than 600 seconds', () => {
+        const date = new Date()
+        const now = new Date()
+        now.setMinutes(date.getMinutes() + 20)
+        expect(niceTime(date, {now})).toBe('20 minutes ago')
+    })
+
+    test('should return timeperiod stamp for timeperiod less than 24 hours', () => {
+        const date = new Date()
+        const now = new Date()
+        now.setHours(date.getHours() + 5)
+        expect(niceTime(date, {now})).toBe(`${('0' + date.getHours()).slice(-2)}:${('0' + date.getMinutes()).slice(-2)}`)
+    })
+
+    test('should return the timeperiod stamp and the day for timeperiod less than 24 hours but not on the same day', () => {
+        const date = new Date()
+        const now = new Date()
+        now.setHours(date.getHours() + 22)
+        expect(niceTime(date, {now})).toBe(`Yesterday ${('0' + date.getHours()).slice(-2)}:${('0' + date.getMinutes()).slice(-2)}`)
```

Comment:
The way it is, each test also tests only a single value. When using `Date.now()` they may however test a different value every time you run the tests. This is not great for reliability: tests that may sometimes fail and sometimes pass would drive me crazy. Better to have it be more deterministic, and use less computation. Tests could use different dates to test various cases, but the dates and expected results would be hard-coded.